docs/user/app_overview.md

```diff
@@ -22,3 +24,14 @@ The Nautobot SSoT app builds atop the [DiffSync](https://github.com/networktocod
 * Glenn Matthew (@glennmatthews)
 * Christian Adell (@chadell)
 * Justin Drew (@jdrew82)
+
+## Nautobot Features Used
+
+!!! warning "Developer Note - Remove Me!"
+    What is shown today in the Installed Plugins page in Nautobot. What parts of Nautobot does it interact with, what does it add etc. ?
+
+### Extras
+
+!!! warning "Developer Note - Remove Me!"
+    Custom Fields - things like which CFs are created by this app?
+    Jobs - are jobs, if so, which ones, installed by this app?
```

Comment:
This would need to be polished up, either before or possibly after merging

Reply:
Could you please provide change proposal to update the documentation?

Reply:
Moved to:
https://github.com/nautobot/nautobot-plugin-ssot/pull/255